modin/core/execution/dask/implementations/pandas_on_dask/io/io.py

```diff
@@ -115,14 +115,7 @@ def to_parquet(cls, qc, **kwargs):
             return BaseIO.to_parquet(qc, **kwargs)
 
         output_path = kwargs["path"]
-        if (
-            "storage_options" in kwargs
-            and kwargs["storage_options"] is not None
-            and "client_kwargs" in kwargs["storage_options"]
-        ):
-            client_kwargs = kwargs["storage_options"]["client_kwargs"]
-        else:
-            client_kwargs = {}
+        client_kwargs = kwargs.get("storage_options", {}).get("client_kwargs", {})
```

Comment:
my suggestion was slightly different, handling case when `storage_options` was specified as `None`, and your change fails in the CI
```suggestion
        client_kwargs = (kwargs.get("storage_options") or {}).get("client_kwargs", {})
```